dev/sscce-helpers.ts

```diff
@@ -0,0 +1,10 @@
+import type { Options, Sequelize } from '../packages/core';
+import * as Support from '../packages/core/test/support';
```

Comment:
Why not import `createSequelizeInstance` directly? Does that give naming conflict issues?

Reply:
Yep! The method this file exports is also called `createSequelizeInstance`